### PR TITLE
Hotfix/ecci 480

### PR DESCRIPTION
**Pull request by AnthonySGeorge**
## Include a summary of what this merge request involves (*)
3 column fix

## Call out any relevant implementation decisions
- Are there any links to back up your chosen methodology?
- Why have you taken the approach?
- Any particular problem areas?
## This PR has been tested in the following browsers
- [ ] Arc
- [ ] Edge
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] IE 11 (Windows)
- [ ] iOS Chrome
- [ ] iOS Safari
- [ ] Android Chrome
- [ ] Android Firefox
- [ ] Android default

### CODE REVIEW

File: composer.lock [hunk 1]
```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "fba4e7b84789ee84bfa3a46b5bba64c8",
```
Comment (Polynya) [inline composer.lock:7]: Is this change necessary? It's not in your last commit so may break the tests

Comment (AnthonySGeorge) [reply composer.lock:7]: I didn't commit the lock file. How do I fix this?

**Comment by Polynya** [comment]
Closing. Check PR 338 instead.
